trunk/examples/concurrent/pthreads/races/regression/atomic_store_unsafe.c

```diff
@@ -0,0 +1,25 @@
+//#Unsafe
+
+/*
+ * Author: Frank Schüssele (schuessf@informatik.uni-freiburg.de)
+ * Date: 2023-12-04
+ */
+
+typedef unsigned long pthread_t;
+
+int x, y;
```

Comment:
Variable `y` is unused and can be omitted:

```suggestion
int x;
```